5g-core-topology/packagevariantset-amf-regional.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: PackageVariantSet
+metadata:
+  name: free5g-amf-regional
+spec:
+  upstream:
+    package:
+      repo: https://github.com/nephio-project/free5gc-packages
+      name: pkg-example-amf-bp
+    revision: v1
+  targets:
+  - repositories:
```

Comment:
I know you copied this from my example...but that example is wrong! This should be:

```
  - repositories:
      matchLabels:
        nephio.org/site-type: regional
```

Same with the others.

Also, in a couple sprints this API will change, I am reimplmenting PVS to a `v1alpha2` that works a little differently (but this will still work, I'll have to do conversion webhooks).

Reply:
Done.

I also added a second type of site called "combined" where the three NFs would run on the same cluster. It could be used for a small deployment of free5g and also it would be good to consider two separate deployments when thinking of cross NF references for example.

Reply:
The labels there are actually map entries, so only one will survive unmarshalling. If you want to match an "or", you need to use `matchExpressions` instead of `matchLabels`. See: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/

Reply:
Specifically: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements I think should work.

Reply:
Thanks John, that's done now.